client/app/telecom/telephony/line/calls/click2Call/add-user/telecom-telephony-line-calls-click2Call-add-user.controller.js

```diff
@@ -57,4 +56,68 @@ angular.module("managerApp").controller("TelecomTelephonyLineClick2CallAddUserCt
         });
     };
 
+    /* ===========================
+    =            BULK            =
+    ============================ */
+
+    self.bulkDatas = {
+        billingAccount: $stateParams.billingAccount,
+        serviceName: $stateParams.serviceName,
+        infos: {
+            name: "click2Call",
+            actions: [{
+                name: "click2CallAddUser",
+                route: "/telephony/{billingAccount}/line/{serviceName}/click2CallUser",
+                method: "POST",
+                params: null
+            }]
+        }
+    };
+
+    self.filterServices = function (services) {
+        var filteredServices = _.filter(services, function (service) {
+            return ["sip", "mgcp"].indexOf(service.featureType) > -1;
+        });
+
+        return _.filter(filteredServices, function (service) {
+            return _.some(service.offers, _.method("includes", "sipfax")) ||
```

Comment:
Why do we need this ? To test for voicefax `featureType` ?

Reply:
It's needed to enable Click2Call action in calls management page, so I guess same tests are mandatory for bulk action